code/common/os.q

```diff
@@ -14,4 +14,5 @@ df:{(`$("/";"\\")[NT]sv -1_v;`$-1#v:("/";"\\")[NT]vs pth(string x;x)[10h=type x]
 run:{system"q ",x}
 kill:{[p]@[(`::p);"\\\\";1];}
 sleep:{x:string x; system("sleep ",x;"timeout /t ",x," >nul")[NT]}
-pthq:{[x] $[10h=type x;ssr [x;"\\";"/"];`$ -1 _ ssr [string (` sv x,`);"\\";"/"]]}
\ No newline at end of file
+pthq:{[x] $[10h=type x;ssr [x;"\\";"/"];`$ -1 _ ssr [string (` sv x,`);"\\";"/"]]}

```

Comment:
I think this could be made a little neater, as it avoids some duplication:
```
{[x] $[10h=type x;ssr;{`$-1_ssr[string` sv x,`;y;z]}][x;"\\";"/"]}
```
May need to hear from @jonnypress first.